### ISSUE TITLE
Multiple "Open in Windows Terminal" opes multiple Windows Terminals rather than tabs

### ISSUE THREAD

**Issue by simonvane** [microsoft/terminal#7071]
# Environment
Microsoft Windows NT 10.0.19041.0
Windows Terminal Version: 1.1.2021.0

# Steps to reproduce
- Right click folder then click "Open in Windows Terminal"
- Repeat

# Expected behavior
I would have expected the second "Open in Windows Terminal" to open another tab in the open Windows Terminal session. That's the joy of tabs, right?

# Actual behavior
Opens a new Windows Terminal session.

**Comment by j4james** [comment]
This is probably dependent on issue #4472.

**Comment by zadjii-msft** [comment]
Yep, this is a combo of #4472 or #2227. I'm gonna leave this open as a setting to enable the shell extension to do this, but know that this is pretty far down the backlog, considering the prerequisite work.

**Comment by siegfriedpammer** [comment]
I hope this issue also covers the use case of using Explorer and typing `wt` in the address bar. I would love to have this feature, now I end up with a gazillion of Windows Terminal instances, having a single one would be much cleaner.

Are there any plans to implement this? What could I do to help with this?

**Comment by zadjii-msft** [comment]
Yep there are plans to implement it - see #4472,  #8135, #5000, [Process Model Improvements](https://github.com/microsoft/terminal/projects/5), #8607. It's what I'm actively working on when I'm not triaging github 😜
